contents/people.yml

```diff
@@ -1,9 +1,18 @@
 -
-  fullname:
+  fullname: Mingye Wang
   username: artoria2e5
-  description:
-  longdesc:
-  rurl:
+  description: Magical catgirl without a focus
+  longdesc: >
```

Comment:
Current newline form generates `caffeine.)\nAccording to`. You may want to use `|` which keeps all literal newlines.